pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[project]
+name = "folksonomy_api"
+version = "0.1.0"
```

Comment:
We are already at  1.1.0, see https://github.com/openfoodfacts/folksonomy_api/releases